src/Validator.php

```diff
@@ -29,18 +29,26 @@ class Validator
      *          ['lengthBetween', 4, 10, 'message'=>'test 2'],
      *      ],
      *  ];.
+     *
+     * @var array<string,array<string|array<string|int|callable>>>
```

Comment:
replace `callable` with `\Closure(xxx, yyy): zzz`

Reply:
I think it's just fine to have `callable` in 4.0.
Looks like in 5.0 it will be more precisely defined and then we can switch to Closure.